app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -90,13 +94,42 @@ class PatientSearchResultItemView(
     addressLabel.text = address.completeAddress
   }
 
-  private fun renderPatientNameAgeAndGender(fullName: String, gender: Gender, dateOfBirth: DateOfBirth) {
+  private fun highlightPatientName(searchQuery: String?, patientSearchResult: PatientSearchResultViewModel, dateOfBirth: DateOfBirth): Name {
+    val canHighlight = searchQuery?.let { searchQuery.isNotBlank() && patientSearchResult.fullName.contains(it, ignoreCase = true) } == true
+    return if (canHighlight) {
+      Name.Highlighted(
+          patientName = showPatientNameAgeAndGender(patientSearchResult.fullName, patientSearchResult.gender, dateOfBirth),
+          highlightStart = patientSearchResult.fullName.indexOf(searchQuery!!, ignoreCase = true),
```

Comment:
We can perhaps extract out this index as a variable since this is being used as part of `highlightEnd` as well?